client/cmd/apps.go

```diff
@@ -18,15 +18,13 @@ import (
 )
 
 // AppCreate creates an app.
+
 func AppCreate(id string, buildpack string, remote string, noRemote bool) error {
+	apptype := "buildpack"
 	c, err := client.New()
-	if err != nil {
-		return err
-	}
```

Comment:
This error handling should be left intact, otherwise we're ignoring something potentially useful.